Doc/README.md

```diff
@@ -131,7 +127,7 @@ information.
 MDANSE started as a fork of [version 3 of the nMOLDYN program](https://github.com/khinsen/nMOLDYN3).
 nMOLDYN was originally developed by Gerald Kneller in 1995 and subsequently also by Konrad Hinsen, Tomasz Rog,
 Krzysztof Murzyn, Slawomir Stachura, and Eric Pellegrini. MDANSE includes most of the code of nMOLDYN3, and also code
-from the libraries [MMTK](https://github.com/khinsen/MMTK) and [ScientificPython](https://github.com/khinsen/ScientificPython),
+from the libraries [](https://github.com/khinsen/) and [ScientificPython](https://github.com/khinsen/ScientificPython),
```

Comment:
This change was clearly NOT meant to happen. The MMTK name has to stay here, as this is the explanation of the _origins_ of MDANSE.